src/period.cpp

```diff
@@ -8,15 +8,14 @@
 #include "nanotime/pseudovector.hpp"
 #include "nanotime/utilities.hpp"
 
-// See comment in <R_Ext/Complex.h> in R>=4.3.0. As of 2024-10-30
-//   clang trunk with -Wmissing-braces requires the double-braced
-//   approach for creating Rcomplex objects.
+// See comment in <R_Ext/Complex.h> in R>=4.3.0 and #134. Old
+//   approach to do `Rcomplex{ re, im }` now gives compiler issue
+//   on some compilers, R versions.
 inline Rcomplex makeComplex(double *re, double *im) {
-#if R_VERSION >= R_Version(4, 3, 0) && !defined(R_LEGACY_RCOMPLEX)
-  return Rcomplex{{*re, *im}};
-#else
-  return Rcomplex{*re, *im};
-#endif
+  Rcomplex ret;
+  ret.r = *re;
+  ret.i = *im;
```

Comment:
You can do the assignment by components when passing the standard way by & reference (possibly const &).

Reply:
resolving comment on earlier commit